doc/man3/SSL_new.pod

```diff
@@ -16,12 +16,16 @@ SSL_dup, SSL_new, SSL_up_ref - create an SSL structure for a connection
 
 SSL_new() creates a new B<SSL> structure which is needed to hold the
 data for a TLS/SSL connection. The new structure inherits the settings
-of the underlying context B<ctx>: connection method,
-options, verification settings, timeout settings. An B<SSL> structure is
-reference counted. Creating an B<SSL> structure for the first time increments
-the reference count. Freeing it (using SSL_free) decrements it. When the
-reference count drops to zero, any memory or resources allocated to the B<SSL>
-structure are freed.
+of the underlying context B<ctx>: connection method, options,
+verification settings, timeout settings. Theses settings are copied in the
+B<SSL> structure, so modifing the original B<SSL_CTX> will not affect already
+created B<SSL> instances. An up-ref'd copy of B<ctx> will be stored inside the
+B<SSL> instance for later retrieval (with SSL_get_SSL_CTX). It is thus safe to
+free the B<ctx> instance before the newly created B<SSL> instance.
+An B<SSL> structure is reference counted. Creating an B<SSL> structure for the
+first time increments the reference count. Freeing it (using SSL_free)
+decrements it. When the reference count drops to zero, any memory or resources
+allocated to the B<SSL> structure are freed.
```

Comment:
I think this should makes it more clear.

```suggestion
of the underlying context B<ctx>: connection method, options,
verification settings, timeout settings. Most of these settings are copied in the
B<SSL> structure, so modifying the original B<SSL_CTX> will not affect already
created B<SSL> instances.

An up-refed copy of B<ctx> is stored inside the B<SSL> instance for internal use.
It can be retrieved with SSL_get_SSL_CTX(). It is thus safe to call
SSL_CTX_free() on the B<ctx> instance before the newly created B<SSL> instance
is destroyed. B<SSL> instances are also reference counted. 
```